### ISSUE TITLE
Exception when selecting greater than or equal to 42 files to be shredded - Windows

### ISSUE THREAD

**Issue by SpyTec** [bleachbit/bleachbit#348]
When selecting greater than or equal to 42 files to be shredded on Windows, it doesn't show the items to be deleted but rather `exception in browse_files()`. 

* BleachBit 2.0
* Windows 8.1

Steps taken:

1. Launch Bleachbit
2. Select `Shred files`
3. Mark 42 files or more to be shredded
4. See error `exception in browse_files()` displayed on right container

Each file is 23 characters long including file extension. Folder path is 22 characters long

**Comment by az0** [comment]
@SpyTec  Thanks for the well-written bug report

I tried it with 50 files with short names, and it did not happen. When I made the filenames longer, I saw
````
exception in browse_files()
Traceback (most recent call last):
  File "\bleachbit\Windows.py", line 97, in browse_files
    Title=title)
error: (12291, 'GetOpenFileNameW', 'No error message is available')
````

**Comment by az0** [comment]
By the way, for now here are  workarounds

1. Delete fewer files in each batch, of course :)
1. Drag files and drop onto application (however, this does not work in administrator mode because of a security limitation in Windows itself)
1. Use the new copy and paste feature in BleachBit 2.2 (available from ci.bleachbit.org)

**Comment by az0** [comment]
This was fixed in commit 721ec925

The problem was the buffer length was 1024 bytes, so I increased it to 10,240 bytes. The number of files will vary depending on the length of the pathnames, and in my test I was able to delete over 100 long filenames with the fix.

In about ten minutes a build with the fix will be posted to [ci.bleachbit.org](https://ci.bleachbit.org/), and it will be version 2.0.721. You can test this or most any newer version there. Let me know how it goes.

**Comment by az0** [comment]
The fix was published in [BleachBit 2.1 beta](https://www.bleachbit.org/news/bleachbit-21-beta). Please try it out, and let me know how it goes. 

Soon the fix will be published also in BleachBit 2.2 final.

**Comment by az0** [comment]
The fix was published in [BleachBit 2.1 beta](https://www.bleachbit.org/news/bleachbit-21-beta). Please try it out, and let me know how it goes. 

Soon the fix will be published also in BleachBit 2.2 final.
